### PR TITLE
Ticket Display

### PR DESCRIPTION
**Pull request by thomasogara**
## Change List
* `/isolate/view-requests/` route will now display a page with an Isolate's currently submitted tickets
* Added Servlet for:
     * Fetching a user's submitted requests and the tickets from those requests as JSON.

### UI:
image

### CODE REVIEW

**Review by kevlartest — APPROVED** [review]

**Review by magurmach — APPROVED** [review]

File: src/main/java/com/google/vinet/servlets/IsolateRequestServlet.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+import com.google.vinet.data.*;
+import java.io.*;
+import java.time.*;
+import java.time.format.*;
+import java.util.*;
+import javax.servlet.annotation.*;
+import javax.servlet.http.*;
+
+@WebServlet("/fetch-requests")
+public class IsolateRequestServlet  extends HttpServlet{
```
Comment (gjrb) [inline src/main/java/com/google/vinet/servlets/IsolateRequestServlet.java]: spacing

File: src/main/java/com/google/vinet/servlets/IsolateRequestServlet.java [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    final PreparedQuery results = datastore.prepare(query);
+
+    Gson gson = new Gson();
+
+    List<IsolateRequest> isolateRequests = new LinkedList<>();
+
+    /* An EntityNotFoundException implies the system has stored an isolate's request, but failed to link
+     * that request to the correct Ticket entity in Datastore, the exception must be thrown to allow Google
+     * Cloud Console to observe and log it. It should not be caught by the caller. */
+    try{
```
Comment (gjrb) [inline src/main/java/com/google/vinet/servlets/IsolateRequestServlet.java]: spacing.

File: src/main/java/com/google/vinet/servlets/IsolateRequestServlet.java [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+      for (Entity entity : results.asIterable()){
```
Comment (gjrb) [inline src/main/java/com/google/vinet/servlets/IsolateRequestServlet.java]: spacing

**Review by gjrb — APPROVED** [review]

File: src/main/java/com/google/vinet/servlets/IsolateRequestServlet.java (hunk 3 above)
Comment (thomasogara) [reply src/main/java/com/google/vinet/servlets/IsolateRequestServlet.java]: fixed https://github.com/googleinterns/step226-2020/pull/11/commits/9c66d3fbae782ca235818fcedf82f82f65c0c084

File: src/main/java/com/google/vinet/servlets/IsolateRequestServlet.java (hunk 2 above)
Comment (thomasogara) [reply src/main/java/com/google/vinet/servlets/IsolateRequestServlet.java]: fixed https://github.com/googleinterns/step226-2020/pull/11/commits/9c66d3fbae782ca235818fcedf82f82f65c0c084

File: src/main/java/com/google/vinet/servlets/IsolateRequestServlet.java (hunk 1 above)
Comment (thomasogara) [reply src/main/java/com/google/vinet/servlets/IsolateRequestServlet.java]: fixed https://github.com/googleinterns/step226-2020/pull/11/commits/9c66d3fbae782ca235818fcedf82f82f65c0c084

**Pull request merged**